src/NuGet.Core/NuGet.Configuration/Utility/IReadOnlyCollectionUtility.cs

```diff
@@ -13,20 +13,12 @@ internal static class IReadOnlyCollectionUtility
         internal static IReadOnlyCollection<T> Create<T>(IEqualityComparer<T> comparer, params T[] t)
         {
             var hashSet = new HashSet<T>(t, comparer);
```

Comment:
I almost think we should get rid of this class now that it has no behavior difference.  But it is used in a decent number of places so I'd be fine with leaving it for now, just wanted to throw it out there.